script/tasks/diamondUpdateFacet.sh

```
@@ -105,51 +117,41 @@
+      if isZkEvmNetwork "$NETWORK"; then
+        UPDATE_SCRIPT=$ZK_SCRIPT_PATH
+        echoDebug $UPDATE_SCRIPT
+      else
+        UPDATE_SCRIPT="$DEPLOY_SCRIPT_DIRECTORY""$SCRIPT".s.sol
+      fi
+
+      PRIVATE_KEY=$(getPrivateKey $NETWORK $ENVIRONMENT)
+      echoDebug "Calculating facet cuts for $SCRIPT..."
+
+      if isZkEvmNetwork "$NETWORK"; then
+        RAW_RETURN_DATA=$(FOUNDRY_PROFILE=zksync NO_BROADCAST=true NETWORK=$NETWORK FILE_SUFFIX=$FILE_SUFFIX USE_DEF_DIAMOND=$USE_MUTABLE_DIAMOND PRIVATE_KEY=$PRIVATE_KEY ./foundry-zksync/forge script "$UPDATE_SCRIPT" -f $NETWORK -vvvv --json --skip-simulation --slow --zksync)
```

Shellcheck (0.10.0)

[warning] This assignment is only seen by the forked process.

(SC2097)